app/src/main/java/org/pesmypetcare/mypetcare/services/pet/PetManagerAdapter.java

```diff
@@ -328,8 +325,8 @@ public void deletePetWeight(User user, Pet pet, DateTime dateTime) {
         ExecutorService executorService = Executors.newSingleThreadExecutor();
         executorService.execute(() -> {
             try {
-                ServiceLocator.getInstance().getPetManagerClient().deleteFieldCollectionElement(user.getToken(),
-                        user.getUsername(), pet.getName(), PetData.WEIGHTS, dateTime.toString());
+                ServiceLocator.getInstance().getPetManagerClient().deleteFieldCollectionElement(user.getToken(), user.getUsername(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 132).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5643405)